pom.xml

```diff
@@ -79,6 +79,11 @@
                 <artifactId>adele-init</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
```

Comment:
Could you please also add `adele-order-manager` as a managed dependency here, and furthermore add it to the `adele-application` pom as a dependency please 😜